pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jsoup</groupId>
+            <artifactId>jsoup</artifactId>
+            <version>1.10.2</version>
```

Comment:
is the version is one of latest? We need to use as new as possible to have less bugs and maybe some new features

Reply:
Ok. I have changed  for the latest 1.15.4